documentation-site/components/yard/config/combobox.ts

```diff
@@ -100,6 +100,11 @@ const ComboboxConfig: TConfig = {
       value: undefined,
       type: PropTypes.String,
       description: 'Name attribute.',
+    },
+    inputRef: {
```

Comment:
I was able to provide a ref for the datetime component:

https://github.com/gadget-inc/gadget/blob/main/packages/web/src/components/render/views/form/DateTimePicker.tsx#L30

One of the issues is that the input is nested, so you have to go:

```js
Input: { props: { overrides: { Input: { props: { ref: forwardProps.ref } } } } },
```

Just wanted to double check to see if that would work here.

Reply:
https://github.com/gadget-inc/baseweb/pull/2/files#diff-b31b4cdb8a894111f734f0926cc0bef16c4a9e21633a98629f35288b5840014aR307

Unfortunately overriding it means that the internal ref will never get set and that breaks its behaviour. The internal component needs the ref as well as forwarding it along

Reply:
Ahhh, gotcha. That's a whole lot of sad pandas.